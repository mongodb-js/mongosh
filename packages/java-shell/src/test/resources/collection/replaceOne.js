// before
db.coll.remove({});
db.coll.insertOne({a: 1});
// command
db.coll.replaceOne({a: 1}, {a: 2});
// command
db.coll.find();
