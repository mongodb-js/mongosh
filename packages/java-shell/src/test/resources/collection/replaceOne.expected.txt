AcknowledgedUpdateResult{matchedCount=1, modifiedCount=1, upsertedId=null}
[ { "_id": <ObjectID>, "a": 2 } ]